frontend/src/metabase/components/AppBanner/AppBanner.unit.spec.tsx

```diff
@@ -152,4 +158,168 @@ describe("AppBanner", () => {
       ).toBeInTheDocument();
     });
   });
+
+  describe("TrialBanner", () => {
+    const token = {
+      valid: true,
+      trial: true,
+    };
+
+    const getTokenExpiryDate = (daysRemaining: number) => {
+      return (
+        dayjs()
+          .add(daysRemaining, "days")
+          // Safety buffer - even a second would do but we want to
+          // account for the slower testing environments, like CI.
+          .add(1, "minute")
```

Comment:
The alternative would be to mock `now` in order to "freeze the time". Either way is fine with me.